### PR TITLE
Add relevant links to Kubernetes blog subproject readme

### PR DESCRIPTION
**Pull request by onlydole**
- Add Kubernetes blog link
- Add Slack blog channel link 
- Add formatting for preferred PR label
- Add GitHub team links

### CODE REVIEW

File: sig-docs/blog-subproject/README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ Regular Blog Meeting: Tuesdays at 18:30 UTC (biweekly). [Convert Your Timezone](
 
 ## Contact
 
-- Slack: #kubernetes-docs-blog
+- Slack: [#kubernetes-docs-blog](https://kubernetes.slack.com/messages/CJDHVD54J)
 - Mailing List: [blog@kubernetes.io](mailto:blog@kubernetes.io)
-- Open Community Issues/PRs: area/blog label
-- GitHub Team: @kubernetes/sig-docs-blog-reviews
+- Open Community Issues/PRs: `area/blog` label
```
Comment (spiffxp) [inline sig-docs/blog-subproject/README.md]: nit: which repo?
```suggestion
- Open Community Issues/PRs: [`is:open repo:kubernetes/website label:area/blog`] (https://github.com/issues?q=is%3Aopen+label%3Aarea%2Fblog+repo%3Akubernetes%2Fwebsite)
```

File: sig-docs/blog-subproject/README.md [hunk 2]
```diff
@@ -20,10 +20,10 @@ Regular Blog Meeting: Tuesdays at 18:30 UTC (biweekly). [Convert Your Timezone](
...
+- GitHub Teams: [kubernetes-blog](https://github.com/orgs/kubernetes/teams/kubernetes-blog), [kubernetes-blog-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-blog-maintainers)
```
Comment (spiffxp) [inline sig-docs/blog-subproject/README.md]: nit: which org?
```suggestion
- GitHub Teams: [@kubernetes/kubernetes-blog](https://github.com/orgs/kubernetes/teams/kubernetes-blog), [@kubernetes/kubernetes-blog-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-blog-maintainers)
```

**Comment by onlydole** [comment]
Thank you, @spiffxp! Great suggestions 👍

**Comment by spiffxp** [comment]
/approve
/lgtm

also please take a look at https://github.com/kubernetes/community/pull/3850

**Pull request merged**